client/src/components/Card.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import Badge from 'react-bootstrap/Badge';
+
+type Props = {
```

Comment:
don't understand :D

Reply:
for props and state (here and in other components) we want to ensure that they are "exact", meaning we don't pass “extra” properties for props for example.
we are doing it simply by wrapping the object with `|`.
e.g.:
`type FooT = {| foo: string |};`
instead of `type FooT = { foo: string };`